frontend/src/components/SimpleRange/SimpleRange.js

```diff
@@ -0,0 +1,48 @@
+import { FormattedNumber } from 'react-intl'
+import { styled } from '@mui/material/styles'
+import LinearProgress, { linearProgressClasses } from '@mui/material/LinearProgress'
+import Tooltip from '@mui/material/Tooltip'
+import PropTypes from 'prop-types'
+
+const GradientProgress = styled(LinearProgress)(({ theme, value }) => ({
+  [`& .${linearProgressClasses.bar}`]: {
+    overflow: 'hidden',
+    backgroundColor: 'transparent',
+    '&:before': {
+      content: '""',
+      display: 'block',
+      width: '100%',
+      height: '100%',
+      transition: 'transform .4s linear',
+      transform: `translateX(${100 - value}%)`,
+      background: `linear-gradient(270deg, ${theme.palette.canary.main} 0%, ${theme.palette.chestnutRose.main} 50%, ${theme.palette.ultramarine.main} 100%)`, // eslint-disable-line
+    },
+  },
+}))
+
+const SimpleRange = ({ min, max, value, style, color }) => {
+  const Range = style === 'gradient' ? GradientProgress : LinearProgress
+  const percentage = Math.min(1, (value - min) / (max - min || 1))
+  return (
+    <Tooltip title={<FormattedNumber value={value} />} placement="top">
```

Comment:
We actually don't want tooltips on ranges. Ranges will be for situations where raw values can mislead users. For example, if someone sees GDP Growth Potential is 1.217 they might think that means "121.7% growth", when in truth the number is meaningless by itself. It only makes sense when compared to the whole range.